packages/vite/src/node/plugins/css.ts

```diff
@@ -936,12 +936,7 @@ const scss: SassStylePreprocessor = async (
       }
     })
   }
-  const importer = [internalImporter]
-  if (options.importer) {
-    Array.isArray(options.importer)
-      ? importer.concat(options.importer)
-      : importer.push(options.importer)
-  }
+  const importer = [internalImporter].concat(options.importer!)
```

Comment:
If `options.importer` would be `undefined` here, this would add an `undefined` value to the array :thinking:  
Do we really want this non-null assertion here?

---

Beside that, does this really fix the issue? And if so: why?

Reply:
I was thinking on something like this to fix the issue:
```js
  const importer = [internalImporter]
  if (options.importer) {
    if( Array.isArray(options.importer) ) {
      importer.push(...options.importer)
    }
    else {
      importer.push(options.importer)
    }
  }
```
This could be written in a more compact form but I don't think we will gain in readability.

Reply:
@patak-js So your solution is exactly the same as before but using `push` instead of `concat`?